docs/pages/includes/install-linux.mdx

```diff
@@ -22,12 +22,14 @@
     </TabItem>
     <TabItem label="Amazon Linux 2/RHEL (RPM)">
         ```code
-        $ sudo yum-config-manager --add-repo https://rpm.releases.teleport.dev/teleport.repo
+        # Source variables about OS version
+        $ source /etc/os-release
+        # Add the Teleport YUM repository for v(=teleport.major_version=). You'll need to update this
+        # file for each major release of Teleport.
+        # Note: if using a fork of RHEL/CentOS or Amazon Linux you may need to use '$ID_LIKE'
+        # and the codename your distro was forked from instead of '$ID'
+        $ sudo yum-config-manager --add $(rpm --eval "https://yum.releases.teleport.dev/$ID/$VERSION_ID/Teleport/%{_arch}/stable/v10/teleport.repo")
```

Comment:
```suggestion
        $ sudo yum-config-manager --add $(rpm --eval "https://yum.releases.teleport.dev/$ID/$VERSION_ID/Teleport/%{_arch}/stable/v(=teleport.major_version=)/teleport.repo")
```

Reply:
Also, what's the difference between `--add-repo` flag that was being used here before, and `--add` which is being used now?

Reply:
Effectively no difference. `yum-config-manager` expands `--add` to `--add-repo`. Updated the flag to the full value just in case there is some weird compat issue with one of the older OS versions we support.